.github/workflows/commercial.yml

```diff
@@ -0,0 +1,65 @@
+name: "[Commercial] Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 8AM, Mondays to Fridays
+    - cron: "0 8 * * MON-FRI"
+
+jobs:
+  # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-outputs-for-a-job
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ steps.main.outputs.repos }}
+    steps:
+      - id: main
+        run: |
+          REPOS=(
+            commercial
+            commercial-canaries
+            commercial-playground
+            commercial-request-parser
+            commercial-shared
+            commercial-templates
+            commercial-tools
+            google-admanager-api
+            Prebid.js
```

Comment:
Worth adding elements-manager as well since we might be developing more actively there?